![alt text](images/7.1-9.png)

In [4]:
from helper import *

In [2]:
xBar = 21.45
s = 0.31
xBar,s

(21.45, 0.31)

## $\bar{x} = 21.45g $
## $s = .031g$

# THIS IS NOT RIGHT USING WRONG LOOKUP FOR T TABLE

In [49]:
t = scipy.stats.t.ppf(.9, 27)
x = t * (s / (28 ** .5))
x

0.07696261923486196

# $ t \approx 1.3137029080232316 $

# $ Lower bound = \bar{x} - t (\frac{\sigma}{\sqrt{n}})$
# $ \approx 21.45 - 1.3137029080232316(\frac{.031}{\sqrt{28}})$
# $ \approx 21.45 - 0.07696261923486196 $
# $ \approx 21.373037380765137 $


# THIS IS NOT RIGHT USING WRONG LOOKUP FOR T TABLE


In [48]:
xBar - x

21.373037380765137